sql.txt

```diff
@@ -1,14 +1,25 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+SELECT CustomerName, ContactName, Country FROM Customers;
+SELECT Country FROM Customers ORDER BY Country;
+SELECT CustomerName, ContactName, Country FROM Customers LIMIT 10 OFFSET 10;
 
```

Comment:
The last response," SELECT CustomerName, ContactName, Country FROM Customers LIMIT 10 OFFSET 10; " was correct. When writing future code, don't add more than one select statement.